namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-preprod/resources/rds.tf

```diff
@@ -0,0 +1,57 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "crime_portal_mirror_gateway_rds" {
+  source                       = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.6"
+  cluster_name                 = var.cluster_name
+  cluster_state_bucket         = var.cluster_state_bucket
+  team_name                    = "probation-in-court"
+  business-unit                = "hmpps"
+  application                  = "crime-portal-mirror-gateway"
+  is-production                = "false"
+  performance_insights_enabled = true
+  db_engine_version            = "10"
+  environment-name             = "development"
```

Comment:
This should probably be preproduction, or I've seen `var.environment-name` used elsewhere?

Reply:
yes - I have updated to use the reference